### PR TITLE
add a quick start page

### PR DESCRIPTION
**Pull request by eshaben**
### Description

Creates a single getting started/quick start page for developers

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: builders/get-started/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+---
+title: Quickly Get Started
+description: Everything you need to know to get started developing, deploying, and interacting with smart contracts on Moonbeam.
+---
+
+# Quick Start Guide for Developing on Moonbeam
+
+![API Providers banner](/images/builders/get-started/endpoints/endpoints-banner.png)
```
Comment (eshaben) [inline builders/get-started/quick-start.md:8]: update banner

File: builders/get-started/.pages [hunk 2]
```diff
@@ -1,6 +1,7 @@
 title: Get Started
 nav:
   - index.md
+  - 'Quick Start': 'quick-start.md'
```
Comment (eshaben) [inline builders/get-started/.pages:4]: add index page icon

File: builders/get-started/quick-start.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+    |                                [Moonscan](https://moonbase.moonscan.io/){target=_blank}                                 |    EVM    |                https://moonbase.moonscan.io/                 |
+    |                   [Blockscout](https://moonbase-blockscout.testnet.moonbeam.network/){target=_blank}                    |    EVM    |    https://moonbase-blockscout.testnet.moonbeam.network/     |
+    |                [Expedition](https://moonbeam-explorer.netlify.app/?network=MoonbaseAlpha){target=_blank}                |    EVM    | https://moonbeam-explorer.netlify.app/?network=MoonbaseAlpha |
+    |                                 [Subscan](https://moonbase.subscan.io/){target=_blank}                                  | Substrate |                 https://moonbase.subscan.io/                 |
+    | [Polkadot.js](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fwss.api.moonbase.moonbeam.network#/explorer){target=_blank} | Substrate |           https://polkadot.js.org/apps/#/explorer            |
+
+
+## Moonbeam Accounts {: #moonbeam-accounts }
```
Comment (eshaben) [inline builders/get-started/quick-start.md:94]: maybe just get rid of this section and add something briefly in the introduction

File: builders/get-started/quick-start.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+    | Public WSS URLs |  <pre style="padding-right: 2em">```wss://moonbase-alpha.public.blastapi.io```</pre> <pre style="padding-right: 2em">```wss://moonbeam-alpha.api.onfinality.io/public-ws```</pre> <pre style="padding-right: 2em">```wss://wss.api.moonbase.moonbeam.network```</pre>   |
+
+!!! note
+    You can create your own endpoint suitable for development or production from one of the [supported RPC providers](/builders/get-started/endpoints/#endpoint-providers){target=_blank}.
+
+### Block Explorers {: #explorers }
+
+Moonbeam provides two different kind of explorers: ones to query the Ethereum API, and others dedicated to the Substrate API. All EVM-based transactions are accessible via the Ethereum API wheras the Substrate API can be relied upon for Substrate-native functions such as governance, staking, and some information about EVM-based transactions. For more information on each explorer, please check out the [Block Explorers](/builders/get-started/explorers){target=_blank} page.
+
+=== "Moonbeam"
```
Comment (eshaben) [inline builders/get-started/quick-start.md:65]: move link to the URL column instead of the name (fix block explorers page too and make this a snippet)

**Review by albertov19 — APPROVED** [review]
Looks good!

**Comment by albertov19** [comment]
I did not merged this as some of the Elements in the checklist need to be marked.

**Comment by albertov19** [comment]
@eshaben I will merge because this will be helpful for the EthBogota Hackathon, but please revisit the checklist :)

**Pull request merged**